test/browser_test.dart

```diff
@@ -0,0 +1,38 @@
+@Tags(['browser'])
+
+import 'dart:js';
+import 'package:node_interop/js.dart';
+import 'package:node_interop/util.dart';
+import 'package:test/test.dart';
+
+final sass = context['sass'] as JsObject;
+
+void main() {
+  test('sass object is available', () {
+    expect(sass, isNotNull);
+  });
+  test('compileString() is available', () {
+    expect(sass['compileString'], isNotNull);
+  });
+  test('compileStringAsync() is available', () {
+    expect(sass['compileStringAsync'], isNotNull);
+  });
+  test('compile() is not available', () {
+    expect(sass['compile'], isNull);
+  });
+  test('render() is not available', () {
+    expect(sass['render'], isNull);
+  });
+  test('renderSync() is not available', () {
+    expect(sass['renderSync'], isNull);
+  });
+  test('compileString() produces output', () {
+    var result = sass.callMethod('compileString', ['foo {bar: baz}']);
+    expect(result['css'], equals('foo {\n  bar: baz;\n}'));
+  });
+  test('compileStringAsync() produces output', () async {
+    var result = sass.callMethod('compileStringAsync', ['foo {bar: baz}']);
+    result = await promiseToFuture<JsObject>(result as Promise);
```

Comment:
This is failing with `TypeError: Instance of 'JsObject': type 'JsObject' is not a subtype of type 'Promise'`. Not sure what to do

Reply:
Let's declare the API for the `compile*` methods and call them directly instead of `callMethod`. Additionally figure out if we can get the spec tests to run in the browser, if that doesn't happen let's test a limited surface of the API in this repo (including basic compilation and forbidden methods)

Reply:
Need to make JS API tests run in the browser, both in this repo and in `sass/sass-spec`